upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -91,6 +91,7 @@ var TerraformCloudProviders = []kops.CloudProviderID{
 	kops.CloudProviderAWS,
 	kops.CloudProviderGCE,
 	kops.CloudProviderHetzner,
+	kops.CloudProviderDO,
```

Comment:
I'm wondering if we should guard the terraform support behind a feature flag until it is fully implemented. I think LoadBalancer and VPC still need support?